### DISCUSSION TITLE
The file change status shown in GitExtensions is not as expected.

### DISCUSSION THREAD

**Discussion by Kotsuha** [gitextensions/gitextensions#11150] [Q&A]
![Combined](https://github.com/gitextensions/gitextensions/assets/10438710/7691e390-02dd-4f2a-b580-8e7f708a2848)

This is the same commit.
TortoiseGit shows that the first file change status is "Added".
GitExtensions shows that the first file change status is "Rename and Modified".

The correct status should be "Added".

After [some googling](https://stackoverflow.com/questions/14832963/how-can-i-control-the-rename-threshold-when-committing-files-under-git), I learned that this information is not stored in the commit. It's determined when you want to see the history.

Is there a way for me to adjust this in GitExtensions?

- Git Extensions 3.5.4.12724
- Build 65f01f39982ccb121c4df7d4b00b506939553958
- Git 2.31.1.windows.1
- Microsoft Windows NT 10.0.19045.0
- .NET Framework 4.8.4645.0
- DPI 96dpi (no scaling)

**Comment by mstv** [comment]
Maybe
![image](https://github.com/gitextensions/gitextensions/assets/36601201/dd78e61e-a696-4c6c-99ce-8998ddee95ea)

**Comment by gerhardol** [comment]
This is not configurable (and no config in Git what I see, onlt command line). (options above are for file history)
https://git-scm.com/docs/git-diff#Documentation/git-diff.txt---find-renamesltngt
The same (hardcoded) handling for listing the differences between commits, patches, range-diff etc.

The similarity index is default 50% could be configurable, but it will almost always not be what you want anyway (unless you never want to detect renamed files at all).

If an option to detect rename and copy is added, add a behavior section in Diff viewer, include "Use histogram".
I suggest a textbox to enable rename /copy and a percentage setter (100% is exact only)
